<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RelocatePhotosDialog</class>
 <widget class="QDialog" name="RelocatePhotosDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>373</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Relocate photos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QLabel" name="explanationLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If you moved some or all of your photos to a new location and PAL can't locate them anymore, you can use this feature to update the file paths stored in the database.&lt;/p&gt;&lt;p&gt;This tool will update the locations for photos you moved from the first to the second path.&lt;/p&gt;&lt;p&gt;Note: This will work for sub-folders too.&lt;/p&gt;&lt;p&gt;For example, if you moved some photos from &lt;span style=&quot; font-style:italic;&quot;&gt;D:\Media\Photos&lt;/span&gt; to &lt;span style=&quot; font-style:italic;&quot;&gt;E:\Archive\Media\Photos&lt;/span&gt;, you can put &lt;span style=&quot; font-style:italic;&quot;&gt;D:&lt;/span&gt; as the old path and &lt;span style=&quot; font-style:italic;&quot;&gt;E:\Archive&lt;/span&gt; as the new one.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="oldPathLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLineEdit" name="oldPathLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>Old location</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="oldPathBrowseButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="newPathLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLineEdit" name="newPathLineEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="placeholderText">
        <string>New location</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="newPathBrowseButton">
       <property name="text">
        <string>Browse...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>5</number>
     </property>
     <item row="0" column="1">
      <widget class="QProgressBar" name="progressBar"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="abortButton">
       <property name="text">
        <string>Abort</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="feedbackLabel"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bottomButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
